README.adoc

```diff
@@ -225,10 +236,7 @@ $ ./refl
 ALL GOOD!
 ----
 
-NOTE: As an example, prior versions of Clojure's own `clojure.stacktrace` made use of reflection (see https://clojure.atlassian.net/browse/CLJ-2502[JIRA CLJ-2502]).
-But this has been addressed via type hints.
-
-Enable or disable the `*warn-on-reflection*` depending on the alias, the following methods are available for each tool.
+To enable or disable `*warn-on-reflection*`, the following methods are available for each tool.
```

Comment:
Not introduced by you, but 

```
`*warn-on-reflection*`
```

in adoc will be bolded as <code>**warn-on-reflection**</code> due to the `*`. One way to escape: 

```
`\*warn-on-reflection*`
```